_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -1241,13 +1240,14 @@ entries:
                  - "202204.0"
 
           - title: "UI components library"
+            url: /docs/marketplace/dev/front-end/ui-components-library/
             include_versions:
             - "202204.0"
             - "202212.0"
             nested:
 
               - title: "Actions"
-                url: /docs/marketplace/dev/front-end/ui-components-library/actions/actions-close-drawer.html
+                url: /docs/marketplace/dev/front-end/ui-components-library/actions/
```

Comment:
This too

Reply:
same here - https://docs.spryker.com/docs/marketplace/dev/front-end/202204.0/ui-components-library/actions/